WcaOnRails/app/webpacker/components/RegionalOrganizations/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import I18n from '../../lib/i18n';
+
+function DisplayOrganizations({
+  orgs,
+}) {
+  const listOrgs = orgs.map((org) => <p>{org.name}</p>);
+  return listOrgs;
+}
+
+function HasOrganizations({
+  orgs,
+}) {
+  if (orgs) {
+    return (
+      <>
+        <p>{I18n.t('regional_organizations.content')}</p>
+        <div className="organizations-list">
+          <DisplayOrganizations orgs={orgs} />
```

Comment:
You can avoid writing a function for that component. Also, a key is needed for react to properfly handle multiple objects
`{orgs.map((org) => <p key={org.name}>{org.name}</p>)}`

Reply:
Thanks for letting me know. Should I keep the function if I need to write multiple lines? Or can the expression you gave also be expanded into multiple lines?